### PR TITLE
[#485] Fix repositories import syntax to be in Kotlin format for maven Jitpack

### PR DESCRIPTION
**Pull request by lydiasama**
- Close https://github.com/nimblehq/android-templates/issues/485

## What happened 👀

- Update the maven import to be in Kotlin format

## Insight 📝

Ref: https://stackoverflow.com/a/66926254/11393626

## Proof Of Work 📹

The CI should be successful.

### CODE REVIEW

**Review by ryan-conway — APPROVED** [review]

**Review by chornerman — APPROVED** [review]

File: RxJavaTemplate[DEPRECATED]/build.gradle.kts [hunk 1]
```diff
@@ -21,7 +21,7 @@ plugins {
 allprojects {
     repositories {
         google()
-        maven(url = "https://jitpack.io")
+        maven { url = uri("https://www.jitpack.io") }
```
Comment (doannimble) [inline RxJavaTemplate[DEPRECATED]/build.gradle.kts:24]: @lydiasama Since this package is deprecated, I'm not sure we should do the update or not 😄

**Review by doannimble — APPROVED** [review]

File: RxJavaTemplate[DEPRECATED]/build.gradle.kts (hunk 1 above)
Comment (lydiasama) [reply RxJavaTemplate[DEPRECATED]/build.gradle.kts:24]: @doannimble I'll keep updating until @ryan-conway said Stop it! 😂

Comment (lydiasama) [reply RxJavaTemplate[DEPRECATED]/build.gradle.kts:24]: @luongvo I'm not sure if I understand your expectation correctly, but do you have another intention outside of fixing this error by fetching Chucker from MavenCentral?

Comment (luongvo) [reply RxJavaTemplate[DEPRECATED]/build.gradle.kts:24]: @lydiasama No, just my concern about fetching the dependency from the recommended source.

Comment (lydiasama) [reply RxJavaTemplate[DEPRECATED]/build.gradle.kts:24]: @luongvo My concern is if we get rid of maven Jitpack, will it affect to another lib we use?

Comment (lydiasama) [reply RxJavaTemplate[DEPRECATED]/build.gradle.kts:24]: @luongvo @ryan-conway The PR is here. https://github.com/nimblehq/android-templates/pull/487

**Pull request merged**